jaseci_core/jaseci/utils/jsorc.py

```diff
@@ -0,0 +1,180 @@
+        self.app_client = client.ApiClient(self.config)
+        self.app_api = client.AppsV1Api(self.app_client)
+
+    def get_pod_list(self):
+        ret = self.api_instance.list_pod_for_all_namespaces(watch=False)
+        res = []
+        for i in ret.items:
+            res.append({"namespace": i.metadata.namespace, "name": i.metadata.name})
+        return res
+
+    def get_deployment_list(self):
+        ret = self.app_api.list_deployment_for_all_namespaces(watch=False)
+        res = []
+        for i in ret.items:
+            res.append({"namespace": i.metadata.namespace, "name": i.metadata.name})
+        return res
+
+    def create_deployment(self, config: dict, namespace: str = "default"):
+        return self.app_api.create_namespaced_deployment(
+            namespace=namespace, body=config
+        )
+
+    def get_deployment_conf(self, name: str, namespace: str = "default"):
+        api_response = self.app_api.read_namespaced_deployment(
+            name=name, namespace=namespace
+        )
+        return api_response
+
+    def patch_deployment_conf(self, config, name: str, namespace: str = "default"):
+        api_response = self.app_api.patch_namespaced_deployment(
+            name=name, namespace=namespace, body=config
+        )
+
+    def deployment_set_scale(
+        self, name: str, namespace: str = "default", scale: int = 1
+    ):
+        # This is just a small shortcut to set the scale of a deployment
+        conf = self.get_deployment_conf(name, namespace)
+        conf.spec.replicas = scale
+        self.patch_deployment_conf(conf, name, namespace)
+
+    def kill_deployment(self, name: str, namespace: str = "default"):
+        try:
+            api_response = self.app_api.delete_namespaced_deployment(
+                name=name, namespace=namespace
+            )
+        except ApiException as e:
+            print("Exception when calling CoreV1Api->delete_namespaced_pod: %s\n" % e)
+
+
+class Monitor:
+    def __init__(self, promonUrl: str, k8sconfig):
+        self.promon = Promon(promonUrl)
+        self.controller = KubeController(k8sconfig)
+
+    def strategy_start_redis(self):
+        deployments = self.controller.get_deployment_list()
+        exsits = False
+        for deployment in deployments:
+            if deployment["name"] == "jaseci-redis":
+                exsits = True
+        if not exsits:
+            logger.info("Creating new jaseci redis")
+            dirpath = os.path.dirname(os.path.realpath(__file__))
+            filepath = os.path.join(dirpath, "jaseci-redis.yaml")
+            self.controller.create_deployment(
+                config=yaml.safe_load(open(filepath, "r"))
+            )
+
+    def strategy_redis_cpu(
+        self, nodeName: str, deploymentNameSpace: str, deploymentName: str
+    ):
+        cpu = self.promon.cpu_utilization_percentage()
+        cpu_usage = cpu[nodeName]
+        print(f"Detect CPU Usage: {cpu_usage}")
```

Comment:
@Gorgeous-Patrick we should use logger instead of print for any logging statements.